client/Encounter/Encounter.ts

```diff
@@ -297,12 +298,18 @@ export class Encounter {
         };
     }
 
-    public LoadSavedEncounter = (savedEncounter: SavedEncounter<SavedCombatant>) => {
+    public LoadSavedEncounter = (savedEncounter: SavedEncounter<SavedCombatant>, autosavedEncounter = false) => {
         const savedEncounterIsActive = !!savedEncounter.ActiveCombatantId;
         const currentEncounterIsActive = this.State() == "active";
 
         savedEncounter.Combatants.forEach(c => {
             const combatant = this.AddCombatantFromStatBlock(c.StatBlock, null, c);
+            if (!autosavedEncounter) {
+                let name = combatant.StatBlock().Name;
+                let counts = combatantCountsByName(name, this.CombatantCountsByName(), name);
+                combatant.IndexLabel = counts[name];
```

Comment:
If I save an encounter in progress with this initiative order:
`Goblin 2`
`Goblin 1`
Will it swap the labels when I load it?

This might be OK, but it's worth noting.

Reply:
It does appear to keep the labels, however if I save an encounter in progress and load more than one instance of it the labels are not incrementing. That must follow a different workflow. Should the expected behavior be the same in that case, or is this different because it's an encounter in progress?

Reply:
Nope, this is the workflow- the `savedEncounterIsActive` flag is what checks for it. While we should never assign duplicate index labels to combatants when they are loaded (in any workflow), we should preserve the index labels on the saved encounter's combatants if possible.

Reply:
Oh, my mistake. I tested that on development, not my feature branch where it's working as expected. However, I did notice that the "Set initiative for" dialogs still show the original index labels so I will work on fixing those.